test/test_profiler.py

```diff
@@ -1282,6 +1288,54 @@ def test_utils_compute_queue_depth_when_no_cuda_events(self):
         basic_evaluation = _utils.BasicEvaluation(prof.profiler)
         self.assertFalse(basic_evaluation.compute_queue_depth())
 
+    def test_utils_compute_idle_time(self):
+        profiler = self.generate_mock_profile()
+        basic_evaluation = _utils.BasicEvaluation(profiler)
+        res = ""
+        for event_key in basic_evaluation.event_keys:
+            res += f"{basic_evaluation.metrics[event_key].idle_time_ns} [{event_key.event.name()}]\n"
+
+        self.assertExpectedInline(
+            res, """\
+0 [CPU (Before cudaLaunchKernel)]
```

Comment:
Why are these idle times zero? Shouldn't it be the whole duration of the kernel?